packages/opentelemetry-types/src/trace/span_context.ts

```diff
@@ -39,10 +38,13 @@ export interface SpanContext {
    * Trace options to propagate.
    *
    * It is represented as 1 byte (bitmap). Bit to represent whether trace is
-   * sampled or not.
+   * sampled or not. When set, the least significant bit documents that the
+   * caller may have recorded trace data. A caller who does not record trace
+   * data out-of-band leaves this flag unset.
+   *
    * SAMPLED_VALUE = 0x1 and NOT_SAMPLED_VALUE = 0x0;
```

Comment:
Would it make sense to have `SAMPLED_VALUE` and `NOT_SAMPLED_VALUE` be exported constants in this file?